static/js/zamboni/static_theme.js

```diff
@@ -224,8 +226,12 @@ $(document).ready(function() {
                     $wizard.find('#submit-describe').submit();
                 } else {
                     data.validation.messages.forEach(function(message) {
-                       $('.errorlist.validator').append($('<li>', {'html': message.message}));
-                       console.error(message);
+                       if (headerImageError) {
+                        $('.header-image-error').append($('<li>', {'html': message.message}));
+                       } else {
+                        $('.theme-name-error').append($('<li>', {'html': message.message}));
```

Comment:
There can be non-name errors thrown too if something unexpected happens - anything the server or linter throws back as an error could be returned.  Though I agree it's most likely going to be something with the name so including it there (so at the top of the form) is better.

Reply:
change `theme-name-error` to `general-validation-error` everywhere so future codebase maintainers don't think it's name specific.